README.md

```diff
@@ -9,55 +9,108 @@ http://52.24.131.202:8999/
+The Gusto application is deployed to the Amazon EC2 (IaaS) and is running on t2.micro CentOS 6.5 servers. The delivery of updates to Gusto is being orchestrated by a TeamCity build server which has been established in EC2 and is also being run on a t2.micro server running CentOS 6.5.
 
-http://localhost:8999/stub/
+Gusto has been mande publicly available at the following dns/ip url:
 
-## Unit Testing
-**Local unit testing**
+[http://gusto.strateratech.com:8999/](http://gusto.strateratech.com:8999/)
 
-- once you've downloaded the repo, change directory to the root gusto directory
-- run commands
-	-  ```npm install```
-	-  ```npm run unit-test```
+[http://52.24.131.202:8999/](http://52.24.131.202:8999/)
 
-**TeamCity unit testing**
 
-- TeamCity will be running the testing using the mocha-teamcity-reporter package for output. the command executed by TeamCity is
-	-  ```npm install```
-	-  ```npm run test```
+5) Wrote unit tests for their code.
 
-## Code Coverage
-**Local unit testing**
+Unit tests were written in the mocha framework. Istanbul was used for code coverage metric collection and the mocha-teamcity-reporter was used to feed the test results to our teamcity server. We have accomplished 100% code coverage.
 
-- once you've downloaded the repo, change directory to the root gusto directory
-- run commands
-	-  ```npm install```
-	-  ```npm run coverage```
+[See test-list](DevelopmentPoolEvidence/teamcity/test-list.JPG)
+
+[See code-coverage](DevelopmentPoolEvidence/teamcity/code-coverage.JPG)
+
+[See it all in TeamCity](http://teamcity.strateratech.com:8111/viewLog.html?buildId=105&buildTypeId=Gusto_PreRelease&tab=testsInfo)
+(use the guest link to login)
+
+6.) Set up or used a continuous integration system to automate the running of tests and continuously deployed their code to their IaaS or PaaS provider
+
+Our TeamCity server is available [here](http://teamcity.strateratech.com:8111).
+(use the guest account link to log in)
+
+TeamCity is actively monitoring all feature branches of the Stratera/Gusto GitHub repo on an ongoing basis. Any commits to any branch will trigger a PreRelease build and unit test execution. Code coverage metrics are collected and all of the test result information is made available in TeamCity. The build result is also provided back to GitHub so that any pull requests are identified as safe or unsafe to merge. 
+
+[See project view](DevelopmentPoolEvidence/teamcity/project-overview.JPG)
+
+[See pull request integration](DevelopmentPoolEvidence/github/pr-teamcity.JPG)
+
+Commits to master automatically package gusto and publish it to npmjs.com. A successful publish job will automatically trigger the Deployment job to Production.
+
+7.) 
+Set up or used configuration management
+
+GitHub is used for our source code repository and our code and configuration content is version controlled, peer reviewed, and tracked on a change-by-change basis. We strictly follow the GitHub Flow branch management strategy.
+
```

Comment:
good.  This may be an area that they down grade us on.  Hopefully not, but if they do, we should try to get positive feedback on how we should address in the next opportunity.  Since we are limited on words, a document that describes our process may be the best approach to address our standards and practices.